src/__tests__/unit/operate.test.js

```diff
@@ -0,0 +1,16 @@
+import operate from '../../logic/operate';
+
+describe('Validate operands functionality', () => {
+  test('multiply', () => {
+    const result = operate('1', '2', 'x');
+    expect(result).toBe('2');
+  });
+  test('add', () => {
+    const result = operate('1', '2', '+');
+    expect(result).toBe('3');
+  });
+  test('modular', () => {
+    const result = operate('2', '2', '%');
+    expect(result).toBe('0');
+  });
```

Comment:
- [ ] Write tests for other operators and also test division by zero.

Reply:
>  On it 🏃🏽

Reply:
> Done ✅